packages/analyzer/src/features/analyse-phase/creators/createClassField.js

```diff
@@ -21,11 +21,17 @@ export function createField(node) {
   /**
    * Add TS type
    * @example class Foo { bar: string = ''; }
-   */ 
+   */
   if(node.type) {
     fieldTemplate.type = { text: node.type.getText() }
   }
 
+  if (isWellKnownType(node)) {
```

Comment:
Can we add a comment here explaining that this just handles `as const`?

Reply:
In fact, maybe we create a handler for this?
```js
fieldTemplate = handleAsConst(fieldTemplate, node);
```

That way we can easily reuse it for variables too

Reply:
> Can we add a comment here explaining that this just handles `as const`?

It also handles namespace types or const enums

```ts
x = Y.Zed;
```